capstone/src/main/java/com/step902020/capstone/Event.java

```diff
@@ -4,6 +4,8 @@
 import org.springframework.cloud.gcp.data.datastore.core.mapping.Field;
 import org.springframework.data.annotation.Reference;
 import org.springframework.data.annotation.Id;
+import org.springframework.lang.Nullable;
```

Comment:
I maybe wrong but I don't think this was used anywhere. Is this the solution to the optional boolean issue?